package.json

```diff
@@ -552,6 +552,7 @@
         "lodash": "^4.17.15",
         "moment": "^2.24.0",
         "original-fs": "^1.1.0",
+        "portfinder": "^1.0.25",
```

Comment:
The toolkit already uses a port related npm package - https://www.npmjs.com/package/tcp-port-used

This one looks handy, but it's worth avoiding package adds if possible.

Could tcp-port-used be used to accomplish the same thing, and if so, would it generate a Windows firewall alert?

Reply:
Didn't realize we had a separate package for that. We'd have to do a manual portscan in that case and increment manually. 

For reference: after a firewall rule reset, our debug workflow still produces a Windows Firewall alert for Docker. It appears that not clicking through the rule message doesn't break our integration tests/debugger functionality.